slices/libnl-3-200.yaml

```
@@ -0,0 +1,22 @@
+package: libnl-3-200
+
+essential:
+  - libnl-3-200_copyright
+
+slices:
+  libs:
+    essential:
+      - libc6_libs
+      - libnl-3-200_config
+    contents:
+      /usr/lib/*-linux-*/libnl-3.so.200:
+      /usr/lib/*-linux-*/libnl-3.so.200.26.0:
```

```suggestion
      /usr/lib/*-linux-*/libnl-3.so.200*:
```
Use this pattern to match potential future patch versions.